.env.e2e

```
@@ -24,3 +24,6 @@ SESSION_SECRET=e2e-test-secret-at-least-32-characters-long
 AUTH_BASE_URL=http://localhost:3001
 TRUSTED_ORIGINS=http://localhost:3001,http://localhost:3000
 HMAC_KEY=flutterwave-webhook-comparison-e2e
+
+GOOGLE_MAPS_API_KEY=google-mapa-api-key
+FLIGHTAWARE_API_KEY=flightaware-api-key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)